lib/hasChangedDeep.js

```diff
@@ -15,7 +15,7 @@ function hasChangedDeep(packages, ignore = []) {
 			// 1. Any local dep package itself has changed
 			if (p._nextType) return true;
 			// 2. Any local dep package has local deps that have changed.
-			else if (hasChangedDeep(p._localDeps, [...ignore, ...packages])) return true;
+			else if (p._localDeps && hasChangedDeep(p._localDeps, [...ignore, ...packages])) return true;
```

Comment:
I came across this bug originally when doing a spike into why our API usage of multi-semantic-release didn't work, essentially, `p._localDeps` can be undefined, as far as I can tell, so this check prevents recursion which would result in an error.

Reply:
Good catch. I've moved `localDeps` resolver one level up. https://github.com/dhoulb/multi-semantic-release/pull/45/commits/7cb62f5e7341bb4c7716f037370305037bdd5824